package/yast2-core.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Jul  4 13:46:48 UTC 2022 - Martin Liška <mliska@suse.cz>
+
+- Fix building with GCC 13 and GCC 12.x.
```

Comment:
Please, add here some reference to a bug, GitHub pull request, feature request or any of those tags that are recognized by the army of OBS bots. For example: bsc#12345.

Reply:
Sure, done.